tools/hardhat-example/README.md

```diff
@@ -3,22 +3,27 @@
 Simple scripts for basic operations like hbars transfer, balance fetching, and contract interactions (deployment and calls).
 
 ## Prerequisite
+
 You must have running:
+
 - JSON-RPC Relay
 
 ## Configuration
 
 Create `.env` file based on `.env.example`
+
 ```
 # Alias accounts keys
 OPERATOR_PRIVATE_KEY=
 RECEIVER_PRIVATE_KEY=
-
+OPERATOR_PRIVATE_KEY_TESTNET=
```

Comment:
See comment below. I don't think we need to add another config variable

Reply:
Removed it.